### PR TITLE
Test

### PR DESCRIPTION
**Pull request by rparrett**
Test comparing to merge base when doing cargo deny checks

### CODE REVIEW

File: .github/workflows/dependencies.yml [hunk 1]
```diff
@@ -29,12 +29,34 @@ jobs:
...
+        run: |
+          set -o history
+          OUT_FILE=/tmp/deny-bans-base
+          cargo deny --format json check bans 2>&1 | jq -r -n '[inputs][] | select(.fields.severity != null) | "\(.fields.severity) \(.fields.message)"' | sort > $OUT_FILE
+      - uses: actions/checkout@v4
+      - name: Cargo update (PR)
+        run: cargo update
+      - name: Check for banned and duplicated dependencies (PR)
+        run: |
+          #!/bin/bash
```
Comment (mockersf) [inline .github/workflows/dependencies.yml:50]: this shouldn't be needed

File: .github/workflows/dependencies.yml [hunk 2]
```diff
@@ -29,12 +29,34 @@ jobs:
...
-        run: cargo deny check bans
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.base_ref }}
+      - name: Cargo update (base)
+        run: cargo update
+      - name: Check for banned and duplicated dependencies (base)
+        run: |
+          set -o history
+          OUT_FILE=/tmp/deny-bans-base
```
Comment (mockersf) [inline .github/workflows/dependencies.yml:43]: could you put the two `OUT_FILE` vars in an `env:` at the start of the job with different names?

File: .github/workflows/dependencies.yml (hunk 1 above)
Comment (rparrett) [reply .github/workflows/dependencies.yml:50]: oops, artifact of local testing

**Comment by rparrett** [comment]
I really wanted to find a convenient way to avoid repeating
```
cargo deny --format json check bans 2>&1 | jq -r -n '[inputs][] | select(.fields.severity != null) | "\(.fields.severity) \(.fields.message)"'
```
without getting into a nightmare of escaping quotes and stuff but ultimately failed

**Comment by mockersf** [comment]
I think you could have a "tooling" step in the job that would output the jq filter to a script, and reuse it in both steps

**Pull request closed without merging**